docs/modules/device-lab-management/pages/deviceConnect/restart-deviceconnect-services.adoc

```diff
@@ -0,0 +1,43 @@
+= Restart deviceConnect services
+:navtitle: Restart deviceConnect services
+
+Sometimes, Hybrid and On-prem customers hosting dedicated devices need to restart deviceConnect services on the Mac mini host to apply configuration changes or software updates. This action can be performed from the Kobiton Portal via the Device Management page.
+
+[NOTE]
+When you restart deviceConnect services, all devices linked to the Mac mini will become inaccessible and any active test sessions will be terminated. Make sure to restart the correct Mac mini and inform all affected users beforehand.
+
+== Prerequisites
+- Gather the internal IP address or hostname of the Mac mini host(s)
+- Kobiton administrator with the ADMIN predefined permission
+
+== Steps to restart deviceConnect services
+Log in to the Kobiton Portal with an administrator account.
+
+include::profile:partial$open-settings.adoc[]
+
+Select the *Device Management* tab.
+
+The *Device Management* page lists all dedicated mobile devices for the org grouped by Mac mini host. Locate the Mac mini host you want to restart and click the corresponding Restart services button.
+
+image:device-lab-management:restart-dc-restart-services-button.PNG[]
+
+In the confirmation modal, choose Restart Services again to proceed.
+
+Any active sessions on devices hosted by that Mac mini will be terminated. During the restart, no actions can be performed on the Mac mini or the connected devices (all the buttons are grayed out). A notification is displayed when the restart completes successfully.
+
+image:device-lab-management:restart-dc-services-restart-successful-popup.PNG[]
+
+After the restart completes, the devices should automatically come back online.
+
+== Impacts on active test sessions during a restart
+When a Mac mini host is restarted using the above method, active sessions on devices hosted by that Mac mini experience the following:
+
+- In the Manual or Mixed session, the user receives a “Device disconnected” error message and the session is terminated with a status of Terminated.
++
+image:device-lab-management:restart-dc-devices-disconnect-popup.PNG[]
+
+- Automation and revisit sessions are also terminated with a status of Terminated.
+
+== Additional notes
+- Mac minis must be restarted one at a time.
```

Comment:
@erincbailey The original content of this line is:

> No support for restarting multiple Mac minis at once.

Saying Mac minis must be restarted one at a time seems to imply that you have to wait for a Mac mini to complete restart before restarting another, which is not true. My original point was that there is no option to select multiple Mac minis and restart them with 1 click.

I suggest changing this sentence to: No option to select multiple Mac minis for restart. What do you think?

Reply:
Aggreed with @erincbailey to remove this line.